scipy/signal/signaltools.py

```diff
@@ -1439,10 +1439,14 @@ def wiener(im, mysize=None, noise=None):
     >>> import numpy as np
     >>> import matplotlib.pyplot as plt
     >>> from pylab import *
```

Comment:
We don't need this line anymore right? That's unnecessary import of the whole namespace